src/lib/viewers/media/_mediaBase.scss

```diff
@@ -24,9 +24,12 @@
 }
 
 .bp-media-container {
+    display: flex;
+    justify-content: center;
     outline: 0 none;
     position: relative;
     user-select: none;  // Prevents copy paste dialog from appearing on mobile
+    width: 100%; // Prevent media controls from overflowing on small screens
```

Comment:
This actually won't work, since it expands the media controls to the full screen width even on small videos.